pom.xml

```diff
@@ -62,6 +62,11 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+	<dependency>
```

Comment:
Shouldn't this dependency be added to jpo-ode-svcs project since other projects don't need it?